lib/buildResponses.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var config = require('../config'),
+    constant = require('../constants'),
+    logger = require('logops');
+
+
+function invokeDashboard(request, options) { // Generic function for sending response configurations
+    request(options, function (err, response, body) {
+        if (err) {
+            logger.error('Request error %s', err);
+        } else {
+            if (response.statusCode = constant.HTTP_RESPONSE_OK) {
+                logger.info('Request done  to url: %s with result:%j', options.url, response.statusCode);
+            }
+            logger.info('done %s to %s', options.method, options.url);
+        }
+    });
+}
+
+function setHttpOptions(message) { // Set http options
+    return {
+        url: config.dashboard.protocol + '://' + config.dashboard.ip + ':' + config.dashboard.port + config.dashboard.path,
+        method: 'POST',
+        json: true,
+        body: message,
+        headers: {
+            'User-Agent': 'request',
+            'content-type': constant.JSON_HEADER
+        }
+    };
+}
+
+function buildNotification() { // creates NGSI v1 notification body
+    logger.info('Creating NGSI notification ...')
+}
+
+
+function createRCMessage() {
+    var request;
```

Comment:
In general requires go better on top of global scope, single line. You can use them as global objects (so request param at invokeDashboard(request, options); is not necessary). (FORGET ABOUT IT. We will do a bigger refactor... continue).